.github/workflows/image-publish-on-push-to-main.yml

```diff
@@ -10,6 +10,13 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  tests_and_sonarcloud:
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-test-sonarcloud.yaml@DBP-332-sonarcloud
```

Comment:
Change when https://github.com/dBildungsplattform/dbp-github-workflows/pull/6 is merged and version tag added.